.github/workflows/build-test.yaml

```diff
@@ -2534,13 +2534,17 @@ jobs:
       - name: run the test
         run: |
           set +e
+          
           echo ${{ secrets.REMOVE_APP_LICENSE }} | base64 -d > license.yaml
+          APP_NAMESPACES=(postgres-test redis-test nginx-test rabbitmq-system)
```

Comment:
let's move this to `env` for the github action step?

Reply:
`env` doesn't support array. workarounds are not clean